linera-chain/src/chain.rs

```diff
@@ -71,6 +71,36 @@ pub static WASM_FUEL_USED_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
     .expect("Counter can be created")
 });
 
+pub static WASM_NUM_READS_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "wasm_num_reads_per_block",
+        "Wasm num reads per block",
```

Comment:
Isn't this line supposed to be a description in English?

Reply:
Yes, done.